danuluma/dannstore
# #161330034 code refactor

#### What does this PR do?
Implement feedback and update readme

#### Description of Task to be completed?
Fix pep8 errors
Update readme
Fix failing tests after refactor

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Run ```python3 -m unittest``` to run tests
* They should all pass

#### What are the relevant pivotal tracker stories?
[#161330034](https://www.pivotaltracker.com/story/show/161330034)
